src/smith_wilson.jl

```diff
@@ -0,0 +1,135 @@
+using LinearAlgebra
+
+export SmithWilsonYield, CalibrationInstruments
+export InstrumentQuotes, ZeroCouponQuotes, SwapQuotes, BulletBondQuotes
+
+"""
+    SmithWilsonYield(ufr,alpha,u,qb)
```

Comment:
I think it might be better to just call this `SmithWilson`. I don't think there's an ambiguity issue, and reads better when imported elsewhere, e.g.:

```julia
import Yields

Yields.SmithWilson(...)
```

Reply:
Also, the literature seems to be so consistent in the use of `α` and Julia make it so easy to enter unicode, I think we should just use `α` instead of `alpha`